pipeline-templates/job/code-build.yml

```diff
@@ -18,6 +18,9 @@ jobs:
       configFilePath: GitVersion.yml
 
   - template: azure-pipelines-templates/build/step/app-build.yml@das-platform-building-blocks
+    parameters:
+          SonarCloud: true
+          SonarCloudProjectKey: SkillsFundingAgency_das-employerdemand-web

```

Comment:
this'll be for the wrong app

Reply:
Re-pushed and added correct one. copy pasta strikes again